app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PostsController < ApplicationController
   before_action :authenticate_user!, only: %i[create destroy]
+  load_and_authorize_resource
```

Comment:
I am unable to sign up in order to log in and check if the authorization works properly.

![image](https://user-images.githubusercontent.com/38088818/187927283-a4891fd1-d297-41b6-83f9-05276d9c8b93.png)

- Kindly look into your registration so that we can be able to log in and test for authorization.

Reply:
I have fixed it. Please check again